libethereum/BlockChain.h

```diff
@@ -118,8 +101,17 @@ class BlockChain
     void process();
 
     /// Sync the chain with any incoming blocks. All blocks should, if processed in order.
-    /// @returns fresh blocks, dead blocks and true iff there are additional blocks to be processed waiting.
-    std::tuple<ImportRoute, bool, unsigned> sync(BlockQueue& _bq, OverlayDB const& _stateDB, unsigned _max);
+    /// @returns tuple with three members - the first (ImportRoute) contains hashes of fresh blocks
+    /// and dead blocks as well as a list of imported transactions. The second is a bool which is
+    /// true iff there are additional blocks to be processed. The third is the imported block count.
+    std::tuple<ImportRoute, bool, unsigned> sync(
```

Comment:
Is this still used?

Reply:
@gumb0 : Yes, this overload is still called in a couple of places (which is why I left it in 😀 )...I forget the exact locations, I can dig them up if you're interested.